examples/ecs/query_bundle.rs

```diff
@@ -0,0 +1,69 @@
+use bevy::{
+    // diagnostic::LogDiagnosticsPlugin,
+    ecs::schedule::RunOnce,
+    log::{LogPlugin, LogSettings},
+    prelude::*,
+};
+
+fn main() {
+    App::build()
+        .insert_resource(LogSettings {
+            level: bevy::log::Level::DEBUG,
+            ..Default::default()
+        })
+        .add_plugin(LogPlugin)
+        .add_plugins(MinimalPlugins)
+        .insert_resource(bevy::app::ScheduleRunnerSettings::run_once())
```

Comment:
I think you can just remove these two lines. The app should just run once by default.

Reply:
You're right